.github/workflows/build.yml

```diff
@@ -57,14 +54,8 @@ jobs:
           ini-values: date.timezone='UTC', pcov.directory=$GITHUB_WORKSPACE, pcov.exclude=#^(?!($GITHUB_WORKSPACE/config/|$GITHUB_WORKSPACE/src/)).*#
           coverage: pcov
 
-      - name: Update composer.
-        run: composer self-update
-
-      - name: Install db and db-sqlite.
-        uses: yiisoft/actions/install-packages@master
-        with:
-          packages: >-
-            ['db', 'db-sqlite']
```

Comment:
What if changes will require changes in db?

Reply:
`composer_root_version: 2.0` doesn't help. `COMPOSER_ROOT_VERSION` has no effect here because dependencies don't depend on `db-migration`.

See result: https://github.com/yiisoft/db-migration/actions/runs/19990071398/job/57329485860?pr=309